lib/node/runner/checker.go

```diff
@@ -660,7 +799,7 @@ func finishOperationPayment(st *storage.LevelDBBackend, tx transaction.Transacti
 	return
 }
 
-func finishProposerTransaction(st *storage.LevelDBBackend, blk block.Block, ptx ballot.ProposerTransaction, log logging.Logger) (err error) {
+func FinishProposerTransaction(st *storage.LevelDBBackend, blk block.Block, ptx ballot.ProposerTransaction, log logging.Logger) (err error) {
```

Comment:
I have a question.. In sync, How do I use this func? sync doesn't know `ptx ballot.ProposerTransaction`... 
 Can we make this struct from `block.Transaction` ?  :-> 
@Charleslee522 @spikeekips